python/MufluxDigiReco.py

```diff
@@ -20,6 +22,33 @@
 #fgeo = ROOT.TFile(geoFile)
 #sGeo = fgeo.FAIRGeom
 
+#function for calculating the strip number from a coordinate
+def StripX(x):
+	
+        #defining constants for rpc properties
+	STRIP_XWIDTH = 0.8625 #internal STRIP V, WIDTH, in cm
+	EXT_STRIP_XWIDTH_L = 0.9625 #nominal (R&L) and Left measured external STRIP V, WIDTH, in cm (beam along z, out from the V plane)
+	EXT_STRIP_XWIDTH_R = 0.86 #measured Right external STRIP V, WIDTH,in cm (beam along z, out from the V plane)
+	V_STRIP_OFF = 0.200
+	NR_VER_STRIPS = 184
+	total_width = (NR_VER_STRIPS - 2) * STRIP_XWIDTH + EXT_STRIP_XWIDTH_L + EXT_STRIP_XWIDTH_R + (NR_VER_STRIPS - 1) * V_STRIP_OFF
+	x_start = (total_width - EXT_STRIP_XWIDTH_R + EXT_STRIP_XWIDTH_L) /2
+        #calculating strip as an integer
+	strip_x = (x_start - EXT_STRIP_XWIDTH_L + 1.5 * STRIP_XWIDTH + V_STRIP_OFF - x)//(STRIP_XWIDTH + V_STRIP_OFF)
+	assert (0 < strip_x > 185),"X strip outside range!"
```

Comment:
This is equivalent to `0 < 185 < strip_x`. I think you'll need to flip the `>` to be a `<`